sebs/aws/aws.py

```
@@ -168,13 +190,18 @@ def _map_architecture(self, architecture: str) -> str:
         # For example, it's 12.x instead of 12.
-        if language == "nodejs":
-            return f"{runtime}.x"
-        return runtime
+        if language == Language.NODEJS:
+            return f"{language_version}.x"
+        elif language == Language.CPP:
+            return "provided.al2"
+        elif language in [Language.PYTHON]:
+            return ("{}{}".format(language, language_version),)
+        else:
+            raise NotImplementedError()
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)